examples/xed-disas-pecoff.cpp

```diff
@@ -181,7 +181,7 @@ class pecoff_reader_t
             disabled_redirection = true;
 #endif
 
-    file_handle_ = CreateFile(input_file_name,
+    file_handle_ = CreateFileA(input_file_name,
```

Comment:
why?

Reply:
In non-ASCII builds, CreateFile will resolve to CreateFileW, expecting a wchar_t * for a filename instead of char * (and leading to a compilation failure).

Reply:
ok... if you want to resubmit with just this change (and an fixed commit message), I'll take it.  The conversion stuff (below) I have covered. Thx for the submission!!